src/amber/router/session/cookie_store.cr

```diff
@@ -36,7 +36,7 @@ module Amber::Router::Session
     forward_missing_to session
 
     def self.build(store, settings)
-      new(store, settings[:key].to_s, settings[:expires].to_i)
+      new(store, settings["key"].to_s, settings["expires"].not_nil!.to_i)
```

Comment:
will the expires `not_nil!` assertion fail if they don't provide a value?  if so, should we consider checking and providing a default of 0?